### PR TITLE
Story/ite 35+ite 37/fe answered polls/fe be integrate answered polls

### PR DESCRIPTION
**Pull request by maniknr-sureify**
(fix):Added CardDate Component for card
(feat): story ITE-35,story ITE-37 completed
image

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -0,0 +1,3 @@
+body{
+    background-color: #E9EDC9;
```
Comment (harshavardhan-sureify) [inline src/App.css:2]: where and where are you using this??use theme??

File: src/components/NavBar.jsx [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+import FormControlLabel from "@mui/material/FormControlLabel";
+import FormGroup from "@mui/material/FormGroup";
+import MenuItem from "@mui/material/MenuItem";
+import Menu from "@mui/material/Menu";
+import logo from "../assets/images/logo.png";
+
+export default function NavBar() {
+    const [auth, setAuth] = React.useState(true);
+    const [anchorEl, setAnchorEl] = React.useState(null);
+
```
Comment (harshavardhan-sureify) [inline src/components/NavBar.jsx:18]: please include screen shots for every feature

File: src/components/routes/AppRoutes.jsx [hunk 3]
```diff
@@ -1,22 +1,43 @@
...
+                    <Route
+                        path="activepolls/:id"
+                        element={<ViewSinglePoll />}
+                    />
+                    <Route path="answeredpolls" element={<AnsweredPolls />} />
+                    <Route
+                        path="answeredpolls/:id"
+                        element={<ViewSinglePoll />}
+                    />
+                    <Route path="*" element={<PageNotFound />} />
```
Comment (harshavardhan-sureify) [inline src/components/routes/AppRoutes.jsx:34]: page not fond should be last of all the routes not only users

File: src/components/NavBar.jsx (hunk 2 above)
Comment (maniknr-sureify) [reply src/components/NavBar.jsx:18]: done

File: src/components/routes/AppRoutes.jsx (hunk 3 above)
Comment (maniknr-sureify) [reply src/components/routes/AppRoutes.jsx:34]: done

**Pull request merged**